modules/imgproc/src/accum.simd.hpp

```diff
@@ -1040,8 +1040,8 @@ void accSqr_simd_(const uchar* src, float* dst, const uchar* mask, int len, int
                 v_src = v_src & v_mask;
                 v_uint16x8 v_src0, v_src1;
                 v_expand(v_src, v_src0, v_src1);
-                v_src0 = v_src0 * v_src0;
-                v_src1 = v_src1 * v_src1;
+                v_src0 = v_mul_wrap(v_src0, v_src0);
```

Comment:
can we instead implement v_mul_expand() for v_uint8 data? I think, it can be implement much more efficiently than 8u to 16u expansion + v_mul_wrap. Besides, this v_mul_expand may appear to be useful for many more operations.

Reply:
Indeed it seems very useful, I used it in here for example
https://github.com/opencv/opencv/blob/068ebe8b4d6061e367798222d025e2971f44f007/modules/imgproc/src/accum.simd.hpp#L1069-L1078
https://github.com/opencv/opencv/blob/e3fb253d56a7be3c10be81673f5902675dcc8e9d/modules/imgproc/src/accum.simd.hpp#L1100-L1103

but in the case you mentiond, we still need to use v_expand + v_mul_wrap since in this case we only square root one vector.
https://github.com/opencv/opencv/blob/e3fb253d56a7be3c10be81673f5902675dcc8e9d/modules/imgproc/src/accum.simd.hpp#L1045-L1049

Reply:
It seems I made a mistake, v_mul_expand for uint8 isn't useful for all theses cases, I'm going to add new universal intrinsic for square root